ticker/src/main/java/com/robinhood/ticker/TickerColumnManager.java

```diff
@@ -32,7 +32,8 @@
  *
  * @author Jin Cao, Robinhood
  */
-class TickerColumnManager extends ArrayList<TickerColumn> {
+class TickerColumnManager {
+    final ArrayList<TickerColumn> tickerColumns = new ArrayList<>();
```

Comment:
It has to be package private since it is referenced in tests

On Wed, Jul 20, 2016 at 12:45 AM Jin Cao <notifications@github.com> wrote:

> In ticker/src/main/java/com/robinhood/ticker/TickerColumnManager.java
> <https://github.com/robinhood/ticker/pull/5#discussion_r71473138>:
>
> > @@ -32,7 +32,8 @@
> >   *
> >   * @author Jin Cao, Robinhood
> >   */
> > -class TickerColumnManager extends ArrayList<TickerColumn> {
> > +class TickerColumnManager {
> > +    final ArrayList<TickerColumn> tickerColumns = new ArrayList<>();
>
> private
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/robinhood/ticker/pull/5/files/4f3b4f67b90ca6edfc4da8daf8cae0cc7a661093#r71473138>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABIOQT4Xci1M8AGqtjEIdC8oVeCxi6KZks5qXcQBgaJpZM4JQc76>
> .
>